.github/workflows/gittuf-rsl-main.yml

```diff
@@ -35,8 +31,8 @@ jobs:
           git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
 
           gittuf_key_path="/tmp/gittuf-key"
-
           echo "$KEY" > $gittuf_key_path
+          chmod 600 $gittuf_key_path # ssh-keygen signer requires this
```

Comment:
Are we using gitsign (Fulcio) or SSH?

Reply:
According to Aditya:

> we're using ssh to sign attestation for the moment, and gitsign to sign the commit itself. once https://github.com/gittuf/gittuf/pull/524 is merged, we can remove the ssh key bits

Reply:
IIUC, the setup-gitsign action is only configuring that every commit is signed, but the key used to sign is the ssh gittuf key.

Reply:
Seeing what [this action](https://github.com/chainguard-dev/actions/blob/main/setup-gitsign/action.yml) does, I am curious as to why we still need lines 27-31.

Reply:
Good eye!

Reply:
>EDIT: Or I suppose, those may be two distinct signing operations: commit signing which uses gpg vs. RSL signing which uses the ssh key.

this is basically it, except commit signing uses GPG and attestation signing uses SSH. Once https://github.com/gittuf/gittuf/pull/524 is released, we can remove the SSH signing piece.

Reply:
>Seeing what this action does, I am curious as to why we still need lines 27-31.

Thanks for catching that, I've removed them now.